zanata-war/src/main/java/org/zanata/ApplicationConfiguration.java

```diff
@@ -76,6 +80,9 @@
     @Getter
     private static final int defaultMaxFilesPerUpload = 100;
 
+    @Getter
+    private static final int defaultAnonymousSessionTimeoutMinutes = 30;
```

Comment:
1. Because there was an existing parameter that measures timeouts in minutes.
2. The Request API requires seconds.
3. If we ever decide to make this a user provided value, makes more sense to define a session timeout in minutes I guess.